### PR TITLE
Objects and Classes tasks

### PR DESCRIPTION
**Pull request by dmitriybelyaev-epam**
Please review my pull request with completed Objects and Classes task.

Thank you,
Dmitriy

### CODE REVIEW

File: tasks/classes/classes.js [hunk 1]
```diff
@@ -6,22 +6,38 @@
...
- *    1 if first greater
- *    0 if equal
- *    -1 if second greater
  */
 //put your code here
 
-const person = //create instance of class person
-const student = //create instance of class student
+class Student extends Person {
+    constructor(grade, firstName, lastName) {
```
Comment (dshaplyko) [inline tasks/classes/classes.js:27]: please use the right order (firstName, lastNale, grade)

**Comment by dmitriybelyaev-epam** [comment]
Hi Dmitriy,

I have watched the webinar again, re-arranged the params and discussed the code with our project's front-end dev. Everything seems ok but still the class is underlined the npm run won't start because of this.

image

**Comment by dshaplyko** [comment]
What version of nodejs are you using? type node -v and let me know

**Comment by dmitriybelyaev-epam** [comment]
> What version of nodejs are you using? type node -v and let me know

image

**Comment by dshaplyko** [comment]
Maybe C in 'class' word is Cyrillic or you forgot to close a curly bracket somewhere at the top.

File: tasks/classes/classes.js [hunk 2]
```diff
@@ -6,22 +6,38 @@
...
 
+class Person {
+    constructor(firstName, lastName) {
+        this.firstName = firstName,
+            this.lastName = lastName;
+    }
+    getFullName() {
+        return this.firstName + " " + this.lastName;
+    }
+
```
Comment (dshaplyko) [inline tasks/classes/classes.js:17]: you did not close properly the class with } bracket.

Comment (dmitriybelyaev-epam) [reply tasks/classes/classes.js:17]: Thanks a lot Dima! Notes taken

image

Comment (dmitriybelyaev-epam) [reply tasks/classes/classes.js:17]: Please approve

**Comment by dshaplyko** [comment]
Approved, please merge it.

**Pull request merged**